### ISSUE TITLE
wait_condition on kubernetes.core.k8s_info causing fail_json error

### ISSUE THREAD

**Issue by sachintanna-lab** [ansible-collections/kubernetes.core#433]
We are utilizing the kubernetes.core.k8s_info module to confirm the successful installation of the ClusterServiceVersion (Operator) we are installing. However, we are receiving the following error:

```
AttributeError: 'K8sAnsibleMixin' object has no attribute 'fail_json'
```
We are **not** receiving this error when using kubernetes.core.k8s.

Our Ansible version is 2.10.8.
Our Python version is 3.9.4.
Our kubernetes.core version is 2.1.1 

An example of our implementation is:

```
- name: Check Status of Operator
  kubernetes.core.k8s_info:
    kind: ClusterServiceVersion 
    api_version: operators.coreos.com/v1alpha1
    name: "operator_name" 
    namespace: "test"
    wait: 'yes'
    wait_condition:
      reason: InstallSucceeded
      type: Succeeded
      status: True
```
Using MacOS Monterey.

**Comment by Akasurde** [comment]
@sachintanna-lab Thanks for reporting this issue. I will try to reproduce this.

**Comment by Akasurde** [comment]
@sachintanna-lab Can you please provide `-vvv` output and the stacktrace?

**Comment by gravesm** [comment]
I'm pretty sure this has been fixed in the 2.3.0 release of kubernetes.core. Can you upgrade and see if this fixes your problem?

**Comment by sachintanna-lab** [comment]
Hello,

We have upgraded to v2.3.0 and re-tested and the error has gone.

Many thanks for your help,

Sachin
